releasenotes/notes/2.0/fix-cached-params-update-4d2814b698fa76b4.yaml

```
@@ -1,8 +1,8 @@
 ---
 fixes:
   - |
-    Fixed a bug in :meth:`.QuantumCircuit.assign_parameters`, occurring when assigning parameters
-    to standard gates whose definition has already been triggered. In this case, the new values
+    Fixed a bug in :meth:.QuantumCircuit.assign_parameters, which occurred when assigning parameters 
```

Now I'm actually unsure whether omitting the quotes is not a new feature I'm unaware of -- sorry if it's correct as is! 
```suggestion
    Fixed a bug in :meth:`.QuantumCircuit.assign_parameters`, which occurred when assigning parameters 
```